SR2019/2019-04-23-competition-wrap-up.md

```diff
@@ -0,0 +1,52 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/news/sr2019/sr2019-photo.jpg)
+
+Hi all,
+
+First off, we'd like to thank you for taking part in Student Robotics 2019 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2019-04-09-sr2019-tlc-crowned-champion/)
+
+Congratulation again to our winners:
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College Guernsey](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| Rookie Award          | [Eltham College](https://www.elthamcollege.london/)
+| Committee Award       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| Robot and Team Image  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| Online Presence       | [Collyer's](https://www.collyers.ac.uk/)
+
+## Event Photos
+
+We've collected together all the media together into 1 album. If you wish to use and republish any of the photos in college material, please be sure to attribute them to [Student Robotics](https://studentrobotics.org).
+
+[Album link](https://photos.app.goo.gl/8vtBZiuM8FXSf79Z8)
+
+## Feedback
+
+We'd love to get your feedback on how the competition went for you this year, so we know what we did well, and where we can improve. Please complete the below short, anonymous survey. We also invite you to share this feedback form with your team members, and anyone else who attended the event.
+
+[Send us feedback](https://docs.google.com/forms/d/e/1FAIpQLSfiJj1awnQbZyGQnavTH4HCNBmRvxeXFZc9v2Q-nsGxtXByCQ/viewform?usp=sf_link)
+
+## Outstanding kit
+
+If you still have some of our kit, please securely send it the address listed on [our website](https://studentrobotics.org/contact/), and let us know so we can expect it and collate it together.
+
+If you were given permission to hold on to your kit after the competition, you already know when to get your kits back to us. For all other outstanding kit, please return to us as soon as possible, especially if you were unable to attend the competition.
+
+## Future Events
+
+We hope to run a competition next year and see you all there. We plan to open places around September, but will let you know as soon as they are.
+
+In the mean time, be sure to follow us on Twitter [@StudentRobotics](https://twitter.com/studentrobotics).
```

Comment:
For my understanding: are we planning to actually put anything out on Twitter, or is this just because that's a mechanism we'll use to advertise us opening places?

Reply:
We'll probably advertise places opening on Twitter, yes. It's also just a good forum to keep an eye on what SR are doing in general.